powersimdata/scenario/create.py

```diff
@@ -356,9 +358,9 @@ def set_engine(self, engine):
 
         :param str engine: simulation engine
         """
-        possible = ["REISE.jl"]
+        possible = [FromREISEjl]
```

Comment:
I updated this to `FromREISEjl` because I changed the code here https://github.com/Breakthrough-Energy/PowerSimData/pull/527/files#diff-39ffb11520dc297abc3cccc10da29d4d4a09d27c9655de99a31635c19c054b32R73-R75 to take in `engine` as a class, so I consequently had to update this `set_engine` code to make the possible value a class. If people gave approval for this direction I would have dove further in and changed all downstream dependencies